### PR TITLE
Ns crmscript enums

### PR DESCRIPTION
**Pull request by kronicz**
Great if you can review my changes

### CODE REVIEW

File: Customer.Service.Support/CRMScript/api reference/Classes/NSContactAgent/GetContactList.md [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
 sortOrder: 1804
 keywords: GetContactList(Integer[])
 ---
 
+Gets an array of Contact objects.
 
+
+
+* **contactIds:** The identifiers of the Contact object
 * **Returns:** IntegerArray
```
Comment (SuperOfficeDevNet) [inline Customer.Service.Support/CRMScript/api reference/Classes/NSContactAgent/GetContactList.md:15]: The returns an array of Contact items.
```suggestion
* **Returns:** an array of NSContact.
```

File: Customer.Service.Support/CRMScript/api reference/Classes/NSDocumentAgent/GetDocumentList.md [hunk 2]
```diff
@@ -7,6 +7,7 @@ sortOrder: 2477
 keywords: GetDocumentList(Integer[])
 ---
 
+Gets an array of Document objects.
 
 * **Returns:** IntegerArray
```
Comment (SuperOfficeDevNet) [inline Customer.Service.Support/CRMScript/api reference/Classes/NSDocumentAgent/GetDocumentList.md:12]: Actually it returns an array of NSDocument.
```suggestion
Returns an array of NSDocument.
```

File: Customer.Service.Support/CRMScript/api reference/Classes/NSAlarmData/SetAssignmentStatus.md [hunk 3]
```diff
@@ -10,7 +10,11 @@ keywords: SetAssignmentStatus(Integer)
...
 
-
+* Unknown= 0
+* None=1
+* Assigning=11
+* Seen=12
+* Declined=13
 
 * **AssignmentStatus:** Integer
 * **Returns:** Integer
```
Comment (SuperOfficeDevNet) [inline Customer.Service.Support/CRMScript/api reference/Classes/NSAlarmData/SetAssignmentStatus.md:20]: SetXxx methods do not return anything... 
```suggestion
* **Returns:** Void
```

**Review by SuperOfficeDevNet — CHANGES_REQUESTED** [review]
I have looked through about 25% of these files and notice that the Set methods have 
**Returns:**  Integer
when in fact they don't return anything - they are Void methods.
Recommend changing to:
**Returns:**  Void

I have made a couple other comments, inline specific files, so check those out and modify accordingly, or accept my suggestions.

**Review by AnthonyYates — APPROVED** [review]
Nice submission! Great job!

**Pull request merged**